.github/workflows/shared-publish-to-docker-versioned.yaml

```diff
@@ -77,7 +77,7 @@ jobs:
         uses: IABTechLab/uid2-shared-actions/actions/shared_create_releases@v2
         with:
           is_release: ${{ steps.setup.outputs.is_release }}
-          version_number_input: ${{ inputs.version_number_input }}
+          version_number_input: ${{ inputs.new_version }}
```

Comment:
It looks like `version_number_input` is no longer used - should it be removed from the inputs on this action?